README.md

```diff
@@ -35,6 +36,36 @@ end
 
 Then run `bundle exec rake test`
 
+### Build failure criteria
+
+By default, `ra11y` will cause a build to fail if any errors or warnings are generated. For some users, this might be too strict. `pa11y` allows you to customize the failure criteria with a `Hash`:
+
+```ruby
+options = {
+  failure: {
+    warnings: 10,
+    errors: 10
+  }
+}
+Ra11y::Site.new("./_site", options: options).run
+```
+
+You can also tell `ra11y` only to provide information, but not to affect the build status one way or the other:
+
+```ruby
+options = {
+  inform_only: true
+}
+Ra11y::Site.new("./_site", options: options).run
```

Comment:
I am too, but what other named option would have if the first named option is options? You're essentially created a hash, the first key of which is options, which contains an options hash. 

What you have:
```ruby
Ra11y::Site.new("./_site", { options: { foo: "bar", foo2: "bar2" } })
```

What I'm saying:

```ruby
Ra11y::Site.new("./_site", foo: "bar", foo2: "bar2")
```